frontend/translations/en.po

```diff
@@ -7523,6 +7523,10 @@ msgstr ""
 "Invalid font weight value: use numeric values (100-950) or standard names "
 "(thin, light, regular, bold, etc.) optionally followed by 'Italic'"
 
+#: src/app/main/data/workspace/tokens/errors.cljs:91
+msgid "workspace.tokens.invalid-token-value-typography"
+msgstr "Invalid typography value: this should be a reference value to another composite typography token."
```

Comment:
When you need to add new translation strings, please ask for the official texts in spanish and english are added to the design or the US. We always release texts in both languages.